client/cli/src/runner.rs

```diff
@@ -177,19 +178,25 @@ impl<C: SubstrateCli> Runner<C> {
 
 	/// A helper function that runs a future with tokio and stops if the process receives the signal
 	/// `SIGTERM` or `SIGINT`.
-	pub fn run_subcommand<BU, B, BA, IQ, CL>(self, subcommand: &Subcommand, builder: BU)
-		-> Result<()>
+	pub fn run_subcommand<B, FBA, IQ, FCL, FB, LB, LCL, LBA, SO>(
+		self, subcommand: &Subcommand, full_builder: FB, light_builder: LB,
```

Comment:
Can we please find a better solution for this.

There is stuff that doesn't support light client and I don't want to dance around these type declarations again. (Cumulus)

We had this once for `run_node` and it was horrible.